app/src/main/res/values-nl/strings.xml

```diff
@@ -87,21 +87,21 @@
     <string name="set_as_wallpaper">Als achtergrond instellen</string>
     <string name="set_as_wallpaper_failed">Achtergrond instellen mislukt</string>
     <string name="set_as_wallpaper_with">Achtergrond instellen met:</string>
-    <string name="no_capable_app_found">Geen app gevonden voor instellen achtergrond</string>
+    <string name="no_capable_app_found">Geen app gevonden voor instellen van achtergrond</string>
     <string name="setting_wallpaper">Achtergrond instellen&#8230;</string>
     <string name="wallpaper_set_successfully">Achtergrond is ingesteld</string>
     <string name="portrait_aspect_ratio">Verhouding in portretmodus</string>
     <string name="landscape_aspect_ratio">Verhouding in landschapsmodus</string>
-    <string name="home_screen">Startscherm</string>
-    <string name="lock_screen">Vergrendelingsscherm</string>
-    <string name="home_and_lock_screen">Start- en vergrendelingsscherm</string>
+    <string name="home_screen">Beginscherm</string>
+    <string name="lock_screen">Vergrendelscherm</string>
+    <string name="home_and_lock_screen">Begin- en vergrendelscherm</string>
 
     <!-- Slideshow -->
     <string name="slideshow">Diavoorstelling</string>
-    <string name="interval">Interval (seconden):</string>
+    <string name="interval">Tussenpoos (seconden):</string>
```

Comment:
That's a really ugly word :-P

Reply:
That's how I learned to translate it in my study (I did a study to become an official translator).